src/modules/externalContacts.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Utilities for interacting with PureCloud conversations
```

Comment:
`with PureCloud External Contacts`.  Actually, with the name change inbound, maybe change this to just `with Extrernal Contacts`